app/components/security.js

```diff
@@ -76,7 +98,7 @@ class Security {
         if (userRoles.some(role => authorisedRoles.includes(role))) {
             next();
         } else {
-            logger.error('[ERROR] :: Error authorising user, Role Not Authorised');
+            logger().error('[ERROR] :: Error authorising user, Role Not Authorised');
```

Comment:
Why not use `req.log.error()` here like above?

Reply:
req not passed in so can't create req.log from logger(req.sessionID).